retail/solutions/churn/README.md

```diff
@@ -0,0 +1,112 @@
+# Retail channel churn predictive model
+
+Unified customer profile is one of the key capabilities of Microsoft Cloud for Retail. In some ways, it’s where the data story comes together. So, what does it do? It helps you gain insights across the complete view of a shopper’s journey. 
+
+With unified customer profile, you can gain a 360-degree perspective of the customer in a clear and intuitive way so that you can provide personalized experiences, reveal important opportunities, prevent potential loss, or churn, and improve customer satisfaction. 
+
+Unified customer profile bolsters **unification**. You can: 
+
+- Bring multiple identities together to create a 360 view of the customer through AI-powered identity resolution. 
+
+- Ingest multiple types of data, behaviors, and customer sentiment in real time via more than 500+ built-in connectors. 
+
+It also fosters **unique enrichment**: 
+
+- Gain a 360 view of the customer with proprietary audience intelligence from Microsoft Graph 
+
+- Leverage cross-channel behavior to complete the picture of your end-customer. 
+
+Furthermore, unified customer profile also gives you better access to customer **insights**: 
+
+- Gain more nuanced insights by combining digital analytics with customer profiles to create richer segments, and leverage churn models to **understand churn risk** at a glance. 
+
+- Observe customer progress through each defined step of the journey, quickly identifying obstacles and opportunities. 
+
+- Create custom reports and views based on real-time customer behavior data, leverage built-in web and mobile analytics to predict customer needs. 
+
+
+Microsoft Cloud for Retail Unified customer profile includes in the U.S. an **AI-based churn predictive model**, designed for omnichannel retail and built atop Customer Insights. Create a retail channel churn predictive model to fit your business needs, and gain cross-channel insights into the chance of retail customer churn. Run your company data through this model, training it to improve its predictions and identify the factors that contribute to churn, at the customer level.
+
+## Prerequisites
```

Comment:
I think we should explicitly call out dependency on **D365 Customer Insights** and **MS Cloud for Retail Add-On**. Thoughts?